commander/src/bootstrapping/commands/config/create.ts

```diff
@@ -78,12 +75,11 @@ export default class NetworkConfigCommand extends Command {
 			if (!userResponse.confirm) {
 				this.error('Operation cancelled, config file already present at the desired location');
 			} else {
-				fs.writeJSONSync(resolve(configPath, 'config.json'), JSON.stringify(defaultConfig));
+				fs.writeJSONSync(resolve(configPath, 'config.json'), defaultConfig, { spaces: '' });
 			}
 		} else {
 			fs.mkdirSync(configPath, { recursive: true });
-			// write config in config.json at the proper path
-			fs.writeJSONSync(resolve(configPath, 'config.json'), JSON.stringify(defaultConfig));
+			fs.writeJSONSync(resolve(configPath, 'config.json'), defaultConfig, { spaces: '' });
```

Comment:
Try sing `'\t'` as spaces, that will result in formatted in more readable and formatted file not the one-liner json.